datasets/fashion_mnist/pipelines/fashion_mnist/pipeline.yaml

```diff
@@ -0,0 +1,118 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+resources: ~
+
+dag:
+  airflow_version: 2
+  initialize:
+    dag_id: fashion_mnist
+    default_args:
+      owner: "Google"
+      depends_on_past: False
+      start_date: '2022-06-10'
+    max_active_runs: 1
+    schedule_interval: "@weekly"
+    catchup: False
+    default_view: graph
+
+  tasks:
+    - operator: "KubernetesPodOperator"
+      description: "Task to copy `t10k-images-idx3-ubyte.gz` from FASHION MNIST Database to GCS"
+      args:
+        task_id: "download_and_process_source_zip_file"
```

Comment:
If you're only downloading and uploading the file to GCS. You can use the BashOperator. See [here](https://github.com/GoogleCloudPlatform/public-datasets-pipelines/blob/9aac4519bd732fa903113846b6438ffe2ab77e5c/datasets/covid19_nyt/pipelines/load_from_github/pipeline.yaml#L65-L87) for an example.

Reply:
done @adlersantos